lib/util/detect_safari.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:html';
+
+import 'package:dart_pad/elements/dialog.dart';
+
+const String _message = '''
+<p>
+It looks like you're using a WebKit-based browser (such as Safari). There's
+currently an issue with the way DartPad and WebKit's JavaScript parser interact
+that could cause up to a thirty second delay the first time you execute Flutter
+code in DartPad. This is not an issue with Dart or Flutter itself, and we're
+working with the WebKit team to resolve it.
+</p>
+<p>
+In the meantime, it's possible to avoid the delay by using one of the other
+major browsers, such as Firefox, Edge (dev channel), or Chrome.
```

Comment:
Suggestion: turn these references into links to install pages:
 - https://www.mozilla.org/firefox/new/
 - https://www.microsoftedgeinsider.com/en-au/download
 - https://www.google.com/chrome/

Reply:
good idea, added.